src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -41,6 +41,23 @@ MOI.get(b::NormInfinityBridge{T, F}, ::MOI.ListOfConstraintIndices{F, MOI.Nonneg
 MOI.delete(model::MOI.ModelLike, c::NormInfinityBridge) = MOI.delete(model, c.nn_index)
 
 # Attributes, Bridge acting as a constraint
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintFunction, c::NormInfinityBridge{T, F}) where {T, F}
+    ge = MOIU.eachscalar(MOI.get(model, MOI.ConstraintFunction(), c.nn_index))
+    t = MOIU.operate!(/, T, sum(ge), T(length(ge)))
+    d = div(length(ge), 2)
+    x = MOIU.operate!(-, T, ge[(d + 1):end], ge[1:d])
+    # x = x / T(2)
+    # x = MOIU.operate!(/, T, x, T(2))
```

Comment:
@blegat I need to divide a VAF by 2 but I can't find a way to do this. If you could add the necessary method for operate in this PR (or in master and I will merge in) that would be a great help.